dashboard/src/features/transactions/components/Summary/Summary.scss

```diff
@@ -27,13 +27,15 @@
   background: none;
 }
 
-.odd td {
-  background: $background-emphasis-color;
+td {
```

Comment:
If we're removing `.odd` row styles, we need to also remove it from the markdown. If that's the intention, I'll add an updated the JSX file to this PR.